tests/tests-resources/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.github.ardoco.core</groupId>
+    <artifactId>tests</artifactId>
+    <version>${revision}</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <artifactId>tests-resources</artifactId>
```

Comment:
What does this module do?

Reply:
It is responsible for handling the diagram gold standard stuff. E.g. it parses the gold standard. It also contains the "StageTest" that the test classes of my stages implement (Runs repetitions of the stage to test determinism, provides caching functionality of test results for comparison, etc)